readthedocs/doc_builder/python_environments.py

```diff
@@ -193,12 +193,12 @@ def install_core_requirements(self):
 
         # Use conda for requirements it packages
         requirements = [
-            'sphinx==1.3.5',
+            'sphinx==1.5.3',
```

Comment:
@benjaoming Given the frequent release cycle for Sphinx, semver, as well as the workload on the core team here, I concur with @ericholscher's recommendation to float up to the next major release. Moving to support Sphinx 1.6 would be a deliberate action.